examples_trame/basic/file_viewer.py

```diff
@@ -0,0 +1,87 @@
+import tempfile
+
+from trame.app import get_server
+from trame.ui.vuetify import SinglePageLayout
+from trame.widgets import vuetify
+
+import pyvista as pv
+from pyvista.trame.ui import ui_container
+
+# -----------------------------------------------------------------------------
+# Trame setup
+# -----------------------------------------------------------------------------
+
+pv.OFF_SCREEN = True
+
+server = get_server()
+state, ctrl = server.state, server.controller
+
+state.trame__title = "Contour"
+ctrl.on_server_ready.add(ctrl.view_update)
+
+# -----------------------------------------------------------------------------
+# Callbacks
+# -----------------------------------------------------------------------------
+
+pl = pv.Plotter()
+
+
+@state.change("files")
+def load_client_files(files, **kwargs):
+    if files is None or len(files) == 0:
+        pl.clear_actors()
+        return
+
+    if files and len(files):
+        if not files[0].get("content"):
+            return
+
+        for file in files:
+            print(f'Load {file.get("name")}')
+            bytes = file.get("content")
```

Comment:
that may not work for large files (multi-piece vs single), you should use the helper class [ClientFile](https://github.com/Kitware/trame/blob/master/examples/validation/core/27_file_upload.py).